esb/plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/AbstractMediator.java

```diff
@@ -190,6 +241,35 @@ public void activate() {
 		if (!reversed) {
 			reverse(this);
 		}
+		if (i == 0) {
+			if (ESBDebuggerUtil.getRecentlyAddedMediator() == null
+					&& (!ESBDebuggerUtil.ispageSaveOperationActivated()
```

Comment:
The method naming is not consistent in "ispageSaveOperationActivated", where in others it's "isPage.." ..